modules/ROOT/pages/servicenow/servicenow-reference.adoc

```diff
@@ -164,27 +187,27 @@ Invokes Web Service operations.
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS | A time unit that qualifies the maxIdleTime attribute. |  |
+** DAYS | A time unit that qualifies the maxIdleTime attribute |  | 
 |===
 
 [[SoapOutputPayload]]
-=== SOAP Output Payload
```

Comment:
Why did you change SOAP to Soap? I think it's supposed to be all caps. Please change all occurrences back unless there is a reason for the change.